src/utils/ensure-base-exists.utils.ts

```diff
@@ -41,12 +45,25 @@ export const ensureBaseTestsExists = async ({
   const { owner, repo } = context.repo;
   const baseRef = event.payload.pull_request.base.ref;
 
+  logger.debug(
+    `Debug: ${JSON.stringify({ owner, repo, base, baseRef }, null, 2)}`
+  );
+
+  const currentBaseSha = await getHeadCommitForRef({
+    owner,
+    repo,
+    ref: baseRef,
+    octokit,
+  });
+
+  logger.debug(`Debug: ${JSON.stringify({ currentBaseSha }, null, 2)}`);
+
   const workflowRun = await getOrStartNewWorkflowRun({
     owner,
     repo,
     workflowId,
     ref: baseRef,
-    commitSha: base,
+    commitSha: currentBaseSha ?? base,
```

Comment:
We'll also need to change this https://github.com/alwaysmeticulous/report-diffs-action/blob/main/src/action.ts#L85 right?